notifier/libnotify.go

```diff
@@ -12,18 +14,43 @@ func SetupLibnotifyNotifier(notifiers *sync.Map) {
 	touch := make(chan Message, 10)
 	notifiers.Store("notifier/libnotify", touch)
 
-	if !libnotify.Init("yubikey-touch-detector") {
-		log.Error("Cannot initialize desktop notifications!")
+	conn, err := dbus.SessionBusPrivate()
+	if err != nil {
+		log.Error("Cannot initialize desktop notifications: ", err)
```

Comment:
To ease future debugging, could you please use unique error messages? Something like "Cannot initialize session bus", "Cannot authenticate DBUS connection", etc - whatever makes sense in your opinion.

Reply:
Fixed!